src/BlazorComponents/ChartJS/ChartJsPlugins.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace BlazorComponents.ChartJS
+{
+    public class ChartJsPlugins
+    {
+        /// <summary>
+        /// The data labels plugin (https://github.com/chartjs/chartjs-plugin-datalabels) option object
+        /// </summary>
+        public ChartJsDataLabels Datalabels { get; set; }
```

Comment:
This is ok for now. But my plan for future is that there should be a model or an extensibility endpoint so that people can write pure c# plugins and have that included with the component. I will start an issue and may be we can have more discussion about this over there.

Reply:
I understand. So the plugin embeddings should not go beyond the bounds...